src/controllers/shared/common.resume.submission.controller.ts

```diff
@@ -110,5 +111,15 @@ const setWebApplicationData = (session: Session, appData: ApplicationData, trans
     mapTrustApiReturnModelToWebModel(appData);
   }
 
+  if (isActiveFeature(config.FEATURE_FLAG_ENABLE_PROPERTY_OR_LAND_OWNER_NOC)) {
```

Comment:
Looks like the containing function gets called on the Registration, Update and Remove journeys. Story says only to implement for Registration but given Update/Remove NOC work will be done soon I think this might be ok and is safe.